client/src/details/details.cpp

```diff
@@ -473,3 +483,36 @@ void Details::setItemsTreeModel(ItemsTreeModel *model)
     // keep the member - could be useful later
     mItemsTreeModel = model;
 }
+
+void Details::copyAddressFromGroup(QGroupBox *box)
+{
+    QMap<QString, QString> fieldNames;
+    fillAddressFieldsMap(box, fieldNames);
```

Comment:
Heh, now fieldNames could be the return value instead of an out parameter ;)

But OK, I'll stop nitpicking here ;)

Reply:
I can't see it without fix it :smile: